docs/configuration.md

```diff
@@ -25,6 +25,13 @@ NAME=Name
 MAX_PLAYERS=10
 ```
 
+## Build arguments
+
+**MODS**
```

Comment:
There need to be two spaces after `**MODS**` to force the line-break in Markdown.